src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Commands/ResumeAzureRmAnalysisServicesServer.cs

```diff
@@ -0,0 +1,60 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.AnalysisServices.Models;
+using Microsoft.Azure.Management.Analysis;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.AnalysisServices.Properties;
+using Microsoft.Azure.Management.Analysis.Models;
+
+namespace Microsoft.Azure.Commands.AnalysisServices
+{
+    [Cmdlet(VerbsLifecycle.Resume, "AzureRmAnalysisServicesServer", 
+        SupportsShouldProcess = true),
+     OutputType(typeof(List<AnalysisServicesServer>))]
```

Comment:
Deosn't seem liek there is any output here.  You should consider havign a PassThru parameter and returnign true if execution is successful.